### DISCUSSION TITLE
Convergence history file output for multizone (CHT) simulations

### DISCUSSION THREAD

**Discussion by praharsh09** [su2code/SU2#2122] [Q&A] [answered]
Hi, 
I am Praharsh, a Ph.D. student at TU Delft & VITO, currently working on developing an automated design chain for CHT problems. 

I noticed that the convergence history file for multizone simulations is output with the case name (line 66 in CMultizoneOutput.cpp).
On the other hand, for single zone simulations, the convergence history file name is rightly output using the CONV_FILENAME option from the config file. 

For our design chain, it is ideal to have a consistent output of history file names for both single-zone and multizone simulations.
 
Am I missing some options for MultiZone simulations for writing the history file with a convergence filename option from cfg?
I would be grateful for your help/suggestions in dealing with this.

**Comment by bigfooted** [comment, marked as answer]
Hi Praharsh, 

Yes, it is clear that cmultizoneoutput:66 uses the case name, and that it is not consistent that way with COutput.cpp:74. Could you create a pull request and implement it as in the single zone case? There might be regression tests that depend on the filename so these also have to be checked but otherwise it looks like a small change.

**Reply by praharsh09** [reply]
Hi Nijso,
Thanks for the response! Indeed, it seems to be a small change. I will open a pull request to make the required changes for a consistent output of the history file.
